dev_guide/deployments.adoc

```diff
@@ -381,3 +382,17 @@ Rollbacks revert an application back to a previous deployment and can be
 performed using the REST API or the CLI. See the
 link:../cli_reference/basic_cli_operations.html#deployment-operations[CLI
 Reference] for more details.
+
+== Resizing
+In addition to rollbacks, you can can exercise fine-grained control over
+the number of replicas by using the `osc resize` command.
+For example, the following command sets the replicas in the deployment
+configuration `frontend` to 3.
+
+----
+$ osc resize dc frontend --replicas=3
+----
+
+The number of replicas will also propagate to the desired state and,
```

Comment:
I suppose the "when" is implicit: "when the system gets around to it".  That was my intent behind using "eventually", to express that the propagation is not immediate, but indeterminate (timewise).  After issuing a resize command, if you inspect desired/current fields, you may not see the new value appear for some (unknown but finite If All Goes Well) time.